cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.7
+
+ADD . /go/src/github.com/uber/jaeger
+
+ENTRYPOINT go run /go/src/github.com/uber/jaeger/cmd/standalone/main.go --span-storage.type=memory
+
+EXPOSE 5775 6831 6832 14267 16686
```

Comment:
I made a comment on earlier PRs that we need to define a single `ports.go` file and consolidate all ports used by the backend in one place. The docker image should be able to expose all ports with N-M range.